Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -26,6 +28,88 @@ namespace Microsoft.MixedReality.Toolkit.Tests
 {
     public class PlayModeTestUtilities
     {
+        // Unity's default scene name for a recently created scene
+        const string playModeTestSceneName = "MixedRealityToolkit.PlayModeTestScene";
+
+        /// <summary>
+        /// Creates a play mode test scene, creates an MRTK instance, initializes playspace.
+        /// </summary>
+        public static void Setup()
+        {
+            if (EditorApplication.isPlaying)
+            {
+                bool sceneExists = false;
+                for (int i = 0; i < SceneManager.sceneCount; i++)
+                {
+                    Scene playModeTestScene = SceneManager.GetSceneAt(i);
+                    if (playModeTestScene.name == playModeTestSceneName && playModeTestScene.isLoaded)
+                    {
+                        SceneManager.SetActiveScene(playModeTestScene);
+                        sceneExists = true;
+                    }
+                }
+
+                if (!sceneExists)
+                {
+                    Scene playModeTestScene = SceneManager.CreateScene(playModeTestSceneName);
+                    SceneManager.SetActiveScene(playModeTestScene);
+                }
+            }
+            else
+            {
+                // At the very start of testing, playmode may not be active.
```

Comment:
Did this part appear due to calling this method in `IPrebuildSetup.Setup` ?  I think this method shouldn't be called outside of play mode at all.

Reply:
YES, and fixing that dramatically reduces setup complexity, thanks for calling this out.